sros2/test/sros2/commands/security/verbs/fixtures/client_service_node.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+from rclpy.node import Node
+
+import test_msgs.srv
+
+
+class ClientServiceNode(Node):
```

Comment:
Can these be turned into an actual pytest fixture on a conftest.py somewhere?

Reply:
Short answer is no, because `test_generate_policy*.py` is no longer a `pytest`, but a `pytest`-run `launch` test.